src/utils/logger.js

```diff
@@ -34,6 +35,7 @@ function initLogger() {
           return `${options.timestamp()} ${winstonCfg.colorize(options.level, options.level.toUpperCase())} ${(options.message ? options.message : '')} ${(options.meta && Object.keys(options.meta).length ? `\n\t${JSON.stringify(options.meta)}` : '')}`;
         },
         json: false,
+        maxFiles: '3d',
```

Comment:
let's make this 14 days. give the users enough time to be able to report issues.